<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Operating_Condition</class>
 <widget class="QWidget" name="Operating_Condition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
	border:1px solid white;
	font:16px;
    color: rgb(192,192,192);
}

QPushButton
{
	font:18px;
    color: rgb(192,192,192);
	background-color: rgb(0, 0, 0);
   border:1px solid rgb(255,255,255);
}</string>
  </property>
  <widget class="QLabel" name="LBL_Line1">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>20</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line3">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>71</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line2">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>46</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line4">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>97</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>97</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>71</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>46</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>123</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>174</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line5">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>123</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line6">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>149</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line7">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>174</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>149</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line8">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>200</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line16">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>406</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>277</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num15">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>303</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line15">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>380</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>226</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>329</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>406</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line9">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>226</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num14">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>355</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line10">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>252</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line13">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>329</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line11">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>277</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>252</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line14">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>355</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Line12">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>303</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CurrentSide">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>437</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	border:0px solid white;
	font:16px;
    color: rgb(192,192,192);
}</string>
   </property>
   <property name="text">
    <string>HXD2 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CurrentSideTrain">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>457</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(255,255,255);
image: url(:/images/images/TrainLogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num26">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>260</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>26</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num18">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>54</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num25">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>234</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num30">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>363</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num20">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>105</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num24">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>208</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num31">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>388</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>31</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num29">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>337</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>29</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num23">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>182</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num19">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>79</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>19</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num17">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>28</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num22">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>157</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>22</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num21">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>131</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num28">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>311</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>28</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num27">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>285</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>27</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Num32">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>414</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
